app/components/Sidebar/Sidebar.tsx

```diff
@@ -1,31 +1,73 @@
-import { List, useMantineTheme } from "@pokt-foundation/pocket-blocks"
+import { List, MediaQuery, useMantineTheme } from "@pokt-foundation/pocket-blocks"
+import { useState } from "react"
+import SidebarRightButton from "../Icons/SidebarRightButton"
 import LinksGroup, { LinksGroupProps } from "../LinksGroup/LinksGroup"
+import SidebarLeftButton from "../Icons/SidebarLeftButton"
 
 type SidebarProps = {
   data: LinksGroupProps[]
 }
 
 export function Sidebar({ data }: SidebarProps) {
+  const [isSidebarOpen, setIsSidebarOpen] = useState(false)
```

Comment:
We can replace this with   `useDisclosure()`
See an example here: https://mantine.dev/core/drawer/#change-transition